### PR TITLE
OSDOC-1983 - Move CSI Operator for GCE from TP to GA

### PR DESCRIPTION
**Pull request by lpettyjo**
Moves GCE CSI Operator feature from TP to GA for 4.8.

OSDOCS: https://issues.redhat.com/browse/OSDOCS-1983
JIRA epic: https://issues.redhat.com/browse/STOR-255
Preview: https://deploy-preview-30740--osdocs.netlify.app/openshift-enterprise/latest/storage/container_storage_interface/persistent-storage-csi-gcp-pd.html

@qinpingli @bertinatto PTAL.

### CODE REVIEW

File: storage/container_storage_interface/persistent-storage-csi-gcp-pd.adoc [hunk 1]
```diff
@@ -10,7 +10,6 @@ toc::[]
 {product-title} can provision persistent volumes (PVs) using the Container Storage Interface (CSI) driver for Google Cloud Platform (GCP) persistent disk (PD) storage.
 
 :FeatureName: GCP PD CSI Driver Operator
-include::modules/technology-preview.adoc[leveloffset=+1]
 
 Familiarity with xref:../../storage/understanding-persistent-storage.adoc#understanding-persistent-storage[persistent storage] and xref:../../storage/container_storage_interface/persistent-storage-csi.adoc#persistent-storage-csi[configuring CSI volumes] is recommended when working with a Container Storage Interface (CSI) Operator and driver.
```
Comment (bobfuru) [inline storage/container_storage_interface/persistent-storage-csi-gcp-pd.adoc:15]: @jsafrane A little later in this section (line 24), there is a note for the driver:

> OpenShift Container Platform defaults to using an in-tree, or non-CSI, driver to provision GCP PD storage. This in-tree driver will be removed in a subsequent update of OpenShift Container Platform. Volumes provisioned using the existing in-tree driver are planned for migration to the CSI driver at that time.

Is this ^^ note still valid in 4.8?

Comment (jsafrane) [reply storage/container_storage_interface/persistent-storage-csi-gcp-pd.adoc:15]: Yes, the note is still valid. Current migration date is OCP 4.10, the note is needed until then.

**Comment by jsafrane** [comment]
lgtm

**Comment by qinpingli** [comment]
@chao007 Please help review from QE side, thanks!

**Comment by chao007** [comment]
Deploy the storage class on your OpenShift Container Platform cluster using the oc command:
Example output
AllowVolumeExpansion:  unset
Here should be True

**Comment by lpettyjo** [comment]
This issue seems to be generic (not 4.8-specific). If so, I'd like to address this in a separate PR.

**Comment by chao007** [comment]
@lpettyjo Yes, please update in 4.7

**Comment by lpettyjo** [comment]
> @lpettyjo Yes, please update in 4.7

@chao007 PR for this fix: https://github.com/openshift/openshift-docs/pull/32578

**Comment by bobfuru** [comment]
LGTM

**Pull request merged**